Bylaw 5

ifgc opened the pull request:
Bylaw 5: Game Changers’ Bylaw or the Equitable Practices Bylaw was passed as written on July 16 2020 under the following terms (also stated in section 3 of said bylaw)

Formatting errors may have occurred while formatting for GitHub

Quorum:8/7

Votes: 7 yes/0 no/ 1 abstain 

Status: Passed, Not Ratified 

Should be noted the 1 abstainer still endorsed it, simply chose not to vote for it. 

This bylaw is the FIRST step in creating a more equitable and diverse team.

Any inadvertent changes to Bylaw 4 or the rest of the document we’re unintentional and should be dismissed

ifgc committed:
Bylaw 5

Bylaw 5: Game Changers’ Bylaw or the Equitable Practices Bylaw was passed as written on July 16 2020 under the following terms (also stated in section 3 of said bylaw)

Formatting errors may have occurred while formatting for GitHub

Quorum:8/7

Votes: 7 yes/0 no/ 1 abstain 

Status: Passed, Not Ratified 

Should be noted the 1 abstainer still endorsed it, simply chose not to vote for it. 

This bylaw is the FIRST step in creating a more equitable and diverse team.

Any inadvertent changes to Bylaw 4 or the rest of the document we’re unintentional and should be dismissed

ifgc committed:
Bylaw 5 Amendments and Revotes

Game Changer’s Bylaw Notes:

The first vote was cast as described below:

Quorum was set at 7, 9 votes were cast.

Passed 8 yes, 0 no, 1 abstain but endorsed.

Yes:

Ishan Coutinho 2017-20
Eugene Kim 2019-20
Jacob Berg 2019-22
Tim Warber  2019-21
Alex Fischman 2019-22
Elias Hayne-Gay 2017-20 
Jenelle Ng 2018-21
Eric Daniel 2018-21

No:

N/A

Abstain:

Ethan Phillips 2017-20

Vote 1 Notes: Ethan Phillips abstained from voting on principle of not voting as a graduating senior, however, they do fully endorse this bylaw

Quorum: 9/7
Votes: 8/0/1 
Status: Passed, Not Ratified 
Section 2 Reopened for Amendments prior to ratification by Coach John Luetke and progressed as described below:

The discussion was reopened to see if any amendments needed to be made, as documented by the comment history on this document. Amendments were proposed as suggestions and voted on as stated below.

Quorum for each amendment was set at 7, votes may be cast as yes, no, or abstain

Quorum: 7/7

Votes: Resolved And Closed 08/03/20 (My Birthday! Great present, this bylaw getting discussed and passed 🎁)
Amendments notes: 

The second vote was cast as described below:

Quorum was set at 7, voting is in progress.

(Currently) 6 yes, 0 no, 0 abstain.


Yes
Alex Fischman 2019-22
Ishan Coutinho 2017-20
Athena Crowther 18-21
John Luetke (Coach)
Eugene Kim 18-20
Eric Daniel 2018-21
Elias Haynie-Gay 2017-20

No:
N/A

Abstain:
N/A



Quorum: 7/7
Votes: 7/0/0
Status: Passed and votes closed 08/05/20

johnluetke commented on Club Constitution.md:
```diff
@@ -254,6 +254,65 @@ The following leads are responsible for maintaining the following roles.
   * Have co-managers (design and hardware specialists) lead the engineering department
   * Jointly responsible with hardware specialist for creating a fall curriculum
+  
+   ### Bylaw #5
```
`### Bylaw 5` (Remove the leading spaces)

johnluetke commented on Club Constitution.md:
```diff
@@ -254,6 +254,65 @@ The following leads are responsible for maintaining the following roles.
+
+ * Recognise and acknowledge that we meet and operate on the traditional land of the first people of Seattle, the Duwamish People past and present, and honor with gratitude the land itself and the Duwamish Tribe.
+
+ - Recognises and acknowledge Systemic Racism/Sexism/Discrimination is real and exists.
```
`*` instead of `-`

johnluetke replied:
Recognize

johnluetke commented on Club Constitution.md:
```diff
@@ -254,6 +254,65 @@ The following leads are responsible for maintaining the following roles.
+ * Recognise and acknowledge that we meet and operate on the traditional land of the first people of Seattle, the Duwamish People past and present, and honor with gratitude the land itself and the Duwamish Tribe.
+
+ - Recognises and acknowledge Systemic Racism/Sexism/Discrimination is real and exists.
+   - In the context of this bylaw, Discrimination (capital D) will be used as an umbrella term that includes but is not limited to discrimination based on gender indentity, disability, department, speech/accent, etc.
```
`*` instead of `-`

johnluetke commented on Club Constitution.md:
```diff
@@ -254,6 +254,65 @@ The following leads are responsible for maintaining the following roles.
+ ## Section 1
+ With the passage of this bylaw FIRST WA Team FRC 4180, The Iron Riders, formally:
+
+ * Recognise and acknowledge that we meet and operate on the traditional land of the first people of Seattle, the Duwamish People past and present, and honor with gratitude the land itself and the Duwamish Tribe.
```
Recognize

johnluetke commented on Club Constitution.md:
```diff
@@ -254,6 +254,65 @@ The following leads are responsible for maintaining the following roles.
+ - Recognises and acknowledge Systemic Racism/Sexism/Discrimination is real and exists.
+   - In the context of this bylaw, Discrimination (capital D) will be used as an umbrella term that includes but is not limited to discrimination based on gender indentity, disability, department, speech/accent, etc.
+
+ * Recognises and acknowledge Systemic Racism/Sexism/Discrimination on basis of identity/Discrimination on Basis of Disability extends to and effects Roosevelt High School as an institution, it’s community, the neighborhoods it’s students and staff come from, Seattle, and the USA as a whole.
```
Recognize

johnluetke commented on Club Constitution.md:
```diff
@@ -254,6 +254,65 @@ The following leads are responsible for maintaining the following roles.
+
+ * Recognises and acknowledge Systemic Racism/Sexism/Discrimination on basis of identity/Discrimination on Basis of Disability extends to and effects Roosevelt High School as an institution, it’s community, the neighborhoods it’s students and staff come from, Seattle, and the USA as a whole.
+
+ * Recognises and acknowledges that as a consequence of the above Systemic Racism/Sexism/Bias extends to Education (including but not limited to Science Technology Engineering and Mathematics (STEM) Education), STEM Programs including but limited to FIRST, and to this team FIRST WA Team Iron Riders, FRC 4180.
```
Recognize

johnluetke commented on Club Constitution.md:
```diff
@@ -254,6 +254,65 @@ The following leads are responsible for maintaining the following roles.
+
+ * Recognises and acknowledges that as a consequence of the above Systemic Racism/Sexism/Bias extends to Education (including but not limited to Science Technology Engineering and Mathematics (STEM) Education), STEM Programs including but limited to FIRST, and to this team FIRST WA Team Iron Riders, FRC 4180.
+
+ * Recognise and acknowledge that Systemic Racism/Sexism/Discrimination can take place through both macro and micro aggressions, through both visible and invisible inequities and are very damaging.
```
Recognize

johnluetke commented on Club Constitution.md:
```diff
@@ -254,6 +254,65 @@ The following leads are responsible for maintaining the following roles.
+
+ * Recognise and acknowledge that Systemic Racism/Sexism/Discrimination can take place through both macro and micro aggressions, through both visible and invisible inequities and are very damaging.
+
+ * Recognises and acknowledges that multiple alumni and members experienced Racism,Sexism, and/or Discrimination while on Iron Riders.
```
Recognize

johnluetke commented on Club Constitution.md:
```diff
@@ -254,6 +254,65 @@ The following leads are responsible for maintaining the following roles.
+
+ * Recognises and acknowledges that multiple alumni and members experienced Racism,Sexism, and/or Discrimination while on Iron Riders.
+
+ * Recognises, even publicly, that the Iron Riders need to do everything within their power to address and eliminate Racism, Sexism, and Discriminition within the team and community and replace it with Equity, Diversity, and Inclusion.
```
Recognize

johnluetke commented on Club Constitution.md:
```diff
@@ -254,6 +254,65 @@ The following leads are responsible for maintaining the following roles.
+ ## Section 2
+ This bylaw will require the team and it’s leadership to comply with the following:
+
+ - Require Student Leaders and the Team’s Mentors/Coaches to through and discuss Equity Diversity and Inclusion Training modules. This must be completed before build season starts at the latest. (It should be noted and recommended that doing this before starting the school year will have the most beneficial effect.)
```
`*` instead of `-`

johnluetke commented on Club Constitution.md:
```diff
@@ -254,6 +254,65 @@ The following leads are responsible for maintaining the following roles.
+ This bylaw will require the team and it’s leadership to comply with the following:
+
+ - Require Student Leaders and the Team’s Mentors/Coaches to through and discuss Equity Diversity and Inclusion Training modules. This must be completed before build season starts at the latest. (It should be noted and recommended that doing this before starting the school year will have the most beneficial effect.)
+    - Specifically Require the following modules or equivalent training determined by Mentors/Coaches
```
`*` instead of `-` in all child bullets

johnluetke commented on Club Constitution.md:
```diff
@@ -254,6 +254,65 @@ The following leads are responsible for maintaining the following roles.
+ - Requires Student Leadership to track students who drop out during the school year, and once a month have an agenda item to discuss reasons for drop out and/or reasons why members have been retained.
+
+ - Requires Iron Riders Leadership to maintain a Black Lives Matter (BLM) statement on the Iron Riders website.
+
```
Delete extra line

johnluetke commented on Club Constitution.md:
```diff
@@ -254,6 +254,65 @@ The following leads are responsible for maintaining the following roles.
+
+ - Requires the Iron Riders to be able to accommodate all students equitably to the best of their abilities.
+
+ ## Section 3
```
Remove leading space

johnluetke commented on Club Constitution.md:
```diff
@@ -254,6 +254,65 @@ The following leads are responsible for maintaining the following roles.
+
+ ## Section 3
+This Bylaw shall be voted on by the rules stated in Article 10 Amendment 1 of the Iron Riders Constitution and will be passed by a simple majority open ballot. It will be ratified by approval of a Coach. Quorum for second was set at 7, 7 votes were cast, bylaw passed 7 yes, 0 no, 0 abstain.
+
```
Remove extra line

johnluetke commented:
@Ishanthecoder after addressing the corrections above, can you please write a short, subjective paragraph or two illustrating the historical context that inspired this amendment? I'd like to start capturing the state of the team/community/world in which substantive changes to the constitution like this are proposed and ratified

ifgc committed:
Garammar fixes

I fixed some grammar errors John pointed out that I thought i had fixed but apparently hadn't. it should be good now.

ifgc committed:
More grammar changes

Apparently I committed none of the grammar changes, so now fixed them and am committing them.I m tired, this isn't really helpful comment I'm sorry, but I really just fixed important grammar changes John pointed out.
